lib/demo_web/router.ex

```diff
@@ -1,12 +1,15 @@
 defmodule DemoWeb.Router do
   use DemoWeb, :router
 
+  import DemoWeb.UserAuth
+
   pipeline :browser do
     plug :accepts, ["html"]
     plug :fetch_session
     plug :fetch_flash
     plug :protect_from_forgery
     plug :put_secure_browser_headers
+    plug :fetch_current_user
```

Comment:
@aaronrenner I have renamed this function. Note this is the one we will most likely have to ask the user to add explicitly to his relevant pipelines.

Reply:
@josevalim What should happen when the user runs the generator twice with different `--web` namespaces? I think these imports will collide...

```
mix phx.gen.auth Accounts User Users --web Warehouse
mix phx.gen.auth Accounts User Users --web Sales
```

Is this an issue worth worrying about?

Reply:
Right. I thought about that. Because we don't namespace controllers and the like, they would all conflict. My suggestion is to check if there is a `AppWeb.UserAuth` module and, if there is, do not generate anything.